### DISCUSSION TITLE
新手,pycharm打开找不到引用,如何把kbe那些库导入呢

### DISCUSSION THREAD

**Discussion by charish-cl** [kbengine/kbengine#1541] [Q&A]
![image](https://user-images.githubusercontent.com/61883438/150710485-35d06dba-f5e5-4ed2-9014-e3b1a9a09bf2.png)

**Comment by charish-cl** [comment]
解决了,需要这里下载tips
https://github.com/kbengine/kbengine_docs.git

**Comment by qhf566** [comment]
想找一款成熟的手游产品源码过来我们深度二次开发(全新换套美术，代码也都从新货混淆及优化整一遍)。意向最好是重度的mmo、arpg、角色扮演或动作手游，其他类型的游戏也可以，必须是高品质大厂近5年内做的产品，国内厂商或国外厂商开发的都可以，产品不能在国内运营过的。可以在国外运营过或运行中。准备了足够夸张的米给到你。如朋友那边有可以给推荐一下有米，不墨迹,鹅浩809996677。
